crates/aptos-genesis/src/config.rs

```diff
@@ -70,6 +70,27 @@ impl Layout {
     }
 }
 
+impl Default for Layout {
+    fn default() -> Self {
+        Layout {
+            root_key: None,
+            users: vec![],
+            chain_id: ChainId::test(),
+            allow_new_validators: false,
+            epoch_duration_secs: 86400,
+            is_test: true,
+            min_stake: 0,
+            min_voting_threshold: 0,
+            max_stake: u64::MAX,
+            recurring_lockup_duration_secs: 86400,
+            required_proposer_stake: 0,
+            rewards_apy_percentage: 1,
+            voting_duration_secs: 1,
+            voting_power_increase_limit: 50,
```

Comment:
20

Reply:
FYI changing this might break tests if any of them are using these default values